consensus/utils.go

```diff
@@ -252,3 +253,10 @@ func isSuperMajorityStake(stake *uint256.Int, totalStake *uint256.Int) bool {
 	// Check if 3Cq - 2N - 1 >= 0
 	return superMajorityConditionSum.Sign() >= 0
 }
+
+func isEqualBlockHashes(hash1 BlockHash, hash2 BlockHash) bool {
```

Comment:
we use `.Eq` for the lib implementation of block hash - would nice if they were the same. Not a deal breaker though.

Reply:
We can't really reuse the .Eq function from the lib package here. The lib package needs to import the consensus package, and not the other way around. We could alternatively update hte .Eq function in lib to use this function. I can return to it later on when renaming these interface types and fields during cleanup later.